### ISSUE TITLE
2 pages at once 

### ISSUE THREAD

**Issue by jettsd** [mihonapp/mihon#28]
### Describe your suggested feature

Add the ability to load 2 pages side by side. I read on a device with a wide screen and I like having it be like I'm reading a book both pages side by side.

### Other details

J2k does have a version of this 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by arrtemis** [comment]
i see someone beat me to it. but yeah, having this feature is amazing for a tablet reader like me, and the only reason I have not yet fully transitioned.

**Comment by jettsd** [comment]
> i see someone beat me to it. but yeah, having this feature is amazing for a tablet reader like me, and the only reason I have not yet fully transitioned.

Same for me. I read on my pixel fold and it makes a great manga companion while at work.

**Comment by dan-malprod** [comment]
Good ol' [#149](https://github.com/tachiyomiorg/tachiyomi/issues/149)

**Comment by LLeopz** [comment]
Yes please. TachiyomiJ2K has this but their app isnt nearly as smooth as mihon proper. To add to the post:

This is how it looks:
![Screenshot_20240118_023304_Mihon](https://github.com/mihonapp/mihon/assets/132709353/b2d51d49-afb5-43fa-b2ee-a98b963c6510)

This is how we hope it will look like:
![Screenshot_20240118_023319_TachiyomiJ2K](https://github.com/mihonapp/mihon/assets/132709353/1be75265-9893-4112-8b0c-8d094e66ac9d)

With real double page spreads behaving as they do already. The app would automatically switch to Double Page viewing mode when in landscape.

**Comment by ianeis** [comment]
Just tried J2K for this feature. It's labelled as a beta/experimental feature on J2K but it's still incredible. J2K in general is much more buggy and awkward to use than mihon so it would be great to have in vanilla mihon.

**Comment by Tuxrug** [comment]
> Just tried J2K for this feature. It's labelled as a beta/experimental feature on J2K but it's still incredible. J2K in general is much more buggy and awkward to use than mihon so it would be great to have in vanilla mihon.

I would prefer using Mihon as well. The screenshots above don't really show how important this feature is, plenty of manga have split-page panels.

![Screenshot_20240503-165710_Shonen Jump.jpg](https://github.com/mihonapp/mihon/assets/82621264/4e29e3a3-09db-4818-adad-39d2fb10a813)
![Screenshot_20240503-165654_Shonen Jump.jpg](https://github.com/mihonapp/mihon/assets/82621264/c3a61ac0-2b52-43a8-a667-9fc4201bc962)

**Comment by meijin3** [comment]
Chiming in to express my interest in such a feature. Viz always splits double pages so being able to merge them would be very useful.

**Comment by AntsyLich** [comment]
Locking the comments as people has nothing better to say than "I would also like the feature"

**Comment by AntsyLich** [comment]
Implemented in #3388
